frontend/packages/console-dynamic-plugin-sdk/src/coderefs/coderef-resolver.ts

```diff
@@ -126,3 +126,33 @@ export const resolveCodeRefProperties = async <E extends Extension<P>, P = Exten
 
   return resolvedValues;
 };
+
+/**
+ *  Returns an extension with its resolved `CodeRef` properties.
+ */
+export const resolveExtensionWithCodeRefs = <E extends Extension<P>, P = ExtensionProperties<E>>(
+  ext: E,
```

Comment:
```suggestion
  extension: E,
```

For consistency with above function :smiley: